go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/SUSE/go-patch
```

Comment:
If we called this `github.com/cppforlife/go-patch` we wouldn't need to change all the imports, right? Consumers can just use a `replace github.com/cppforlife/go-patch => github.com/SUSE/go-patch` in their go.mod or something?

… Now that I'm typing all this out I'm not so sure :D

Reply:
I had a similar suggestion for it as well. My suggestion was to clone the upstream and add the SUSE fork as a git remote. This way we don't even need to add go modules support to this PR (which I don't like very much to be done in this same PR).

Reply:
From https://github.com/golang/go/issues/30354:

>`replace` is intended to support local development, not permanent forks: if you need to permanently fork a dependency, your fork must have its own, unique import path.

So I don't think doing `replace` gymnastics is the right way in this case.